RocketChat/Rocket.Chat.ReactNative
# fix: show the thread when it has zero messages

## Proposed changes

tcount when the thread was not started is null, but when it was started it goes from zero to infinity.
This PR corrects that logic.

## Issue(s)	


## How to test or reproduce


- Create a thread
- Send a couple of messages inside the thread
- go back to main channel and verify the thread’s UI.
- Enter the thread and delete every message sent there.
- Repeat step 3.

## Screenshots

Thread with zero messages:
![Screenshot 2023-08-22 at 10 35 22](https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/18687097/681cc7e8-d68c-4f90-8360-469ef4e5c916)

## Types of changes




## Checklist



## Further comments


Jira: [TC-909]


[TC-909]: https://rocketchat.atlassian.net/browse/TC-909?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
